src/main/java/seedu/address/logic/parser/recipe/RecipeAddCommandParser.java

```diff
@@ -102,11 +103,13 @@ RecipeAddIngredientCommand parseAddIngredient(String args) throws ParseException
      * @throws ParseException if the user input does not conform the expected format
      */
     public RecipeAddStepCommand parseAddStep(String args) throws ParseException {
+        Index recipeIndex;
+        Optional<Index> stepIndex;
```

Comment:
maybe we could directly initialise it to Optional.of(null) so that we just need one if statement without the else, like `if (!stepVariable.equals("next")) {...}`

Reply:
okay fix thanks!